File: android/app/src/main/java/me/aksalj/usiuboard/data/BoardFeed.java
package me.aksalj.usiuboard.data;

/**
 * Copyright (c) 2015 Salama AB
 * All rights reserved
 * Contact: aksalj@aksalj.me
 * Website: http://www.aksalj.me
 * <p/>
 * Project : USIU Board
 * File : me.aksalj.usiuboard.data.BoardFeed
 * Date : Feb, 04 2015 2:01 PM
 * Description :
 */
public class BoardFeed {
    public int id;
    public String title;
    public String description;
    public String url;
    public String slug;

    public BoardFeed(int id, String title, String description, String url) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.url = url;
    }
}

File: ios/NoticeBoard/Data/BoardFeed.swift
//
//  BoardFeed.swift
//  NoticeBoard
//
//  Created by Salama AB on 3/19/15.
//  Copyright (c) 2015 Salama AB. All rights reserved.
//

import Foundation

public class BoardFeed: Hashable {
    
    var id: Int
    var title: String
    var description: String
    var url: NSURL
    
    public var hashValue: Int {
        return "HasValue is \(self.id)".hashValue
    }
    
    init(id:Int, title: String, description:String, url: String) {
        self.id = id
        self.title = title
        self.description = description
        self.url = NSURL(string: url)!
    }
}

public func ==(lhs: BoardFeed, rhs: BoardFeed) -> Bool {
    return lhs.hashValue == rhs.hashValue
}